packages/viewer/src/interactions/interactionApi.ts

```diff
@@ -161,7 +164,7 @@ export class InteractionApi {
    */
   public async endInteraction(): Promise<void> {
     if (this.isInteracting()) {
-      this.currentCamera = null;
+      this.currentCamera = undefined;
```

Comment:
what was the difference in the tsconfig that prompted all the changes from null to undefined?

Reply:
The difference is that the ts config of `--strictNullChecks` is now turned on. Would be happy to discuss turning it off, but this was the result of pulling the shared tsconfig.